python/ray/data/dataset.py

```diff
@@ -3394,6 +3394,18 @@ def write_bigquery(
                 overwritten if it exists.
             ray_remote_args: Kwargs passed to ray.remote in the write tasks.
         """
+        if ray_remote_args is None:
+            ray_remote_args = {}
+
+        # Each write task will launch individual remote tasks to write each block
+        # To avoid duplicate block writes, the write task should not be retried
+        if ray_remote_args.get("max_retries", None) != 0:
```

Comment:
If we default to `None`, you'd get a warning by default whenever you use `read_bigquery`, right?

```suggestion
        if ray_remote_args.get("max_retries", 0) != 0:
```